src/test/java/com/wikia/webdriver/testcases/mercurytests/old/widgettests/TwitterTests.java

```diff
@@ -89,12 +89,12 @@ public void MercuryTwitterWidgetTest_004_areLoadedOnFirstVisitDirectlyFromUrl()
     Assertion.assertTrue(widget.areLoaded(), MercuryMessages.INVISIBLE_MSG);
   }
 
-  @Test(groups = "MercuryTwitterWidgetTest_005", enabled = false)
+  @Test(groups = "MercuryTwitterWidgetTest_005")
   public void MercuryTwitterWidgetTest_005_isErrorPresent() {
     init();
 
     widget.createIncorrect(TWITTER_INCORRECT_WIDGET_ARTICLE_NAME);
-    navigate.toPage(TWITTER_INCORRECT_WIDGET_ARTICLE_NAME);
+    navigate.toPage("/wiki/" + TWITTER_INCORRECT_WIDGET_ARTICLE_NAME);
```

Comment:
case above is wrapper for create article method and because all articles starts with `/wiki/` it requires to provide only article name.

the second method is universal, you have to pass whole path